samples/drivers/adc/boards/stm32h735g_disco.overlay

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2021 STMicroelectronics
```

Comment:
Copyright ?

Reply:
I copied the notice from another board's overlay, I'm not familiar with the process for copyright notices. What should I change here?